packages/utils/README.md

```diff
@@ -0,0 +1,15 @@
+## Introduction
+
+This package provides utility functions for Sygma SDK.
```

Comment:
Here as well we might give a little bit more information about the purpose of the utils, that they are interacting with the sygma protocol logic for example getting balances from handlers. I was confused about what is the difference between utils and utils from the core package